doc/lua_api.txt

```diff
@@ -258,13 +258,17 @@ dependency.
 Aliases
 =======
 
-Aliases can be added by using `minetest.register_alias(name, convert_to)` or
-`minetest.register_alias_force(name, convert_to)`.
+Aliases of itemnames can be added by using
+`minetest.register_alias(alias, original_name)` or
+`minetest.register_alias_force(alias, original_name)`.
 
-This converts anything called `name` to `convert_to`.
+This adds an alias `alias` for the item called `original_name`.
+From now on, you can use `alias` to refer to the item `original_name`.
+
+There are no aliases for entitynames.
```

Comment:
`entity names`.
Please don't document lack of features, otherwise the API file size gets out of control.

Reply:
Done.